lmp/jobserv.yml

```diff
@@ -52,6 +52,7 @@ triggers:
               - intel-corei7-64
               - hikey
               - raspberrypi0-wifi
+              - raspberrypi-cm3
```

Comment:
I don't think we can add this here, because we don't have a CI worker for this type.

Reply:
You're right.  I misunderstood the comment for that section.  Fixed.